@types/gql.d.ts

```diff
@@ -45,7 +45,6 @@ type UpdateProjectArgs = {
 };
 
 type ProjectQuery = {
-    _id?: string;
```

Comment:
the _id properties are no longer needed in types since we no longer need to recall project or projectquery from mongoose to update them during the creation processes

Reply:
The `logged_on` field from the schema did not make it here.

Reply:
fixed